hooks/300-remove-pkgs-late.chroot

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -ex
+
+echo "Remove packages we no longer need after postinst/hooks"
+
+# Who also uses perl stuff?
+# pam-auth-update, which we invoke in 200-enable-pwquality
+
+# remove perl from the base
+# perl-base requires --force-remove-essential since its considered an essential package
+#
+# adduser requires --purge
+# following packages depend on adduser (but only during postinst/postrm scripts afaict) 
+# --- wpasupplicant depends on adduser.
+# --- udev depends on adduser.
+# --- openssh-server depends on adduser.
+# --- openssh-client depends on adduser.
+# --- dbus-system-bus-common depends on adduser.
+dpkg --purge --force-depends --force-remove-essential perl-base adduser
+
+# cleanup perl modules left behind by other packages
+find / -type f -name '*.pm' -delete
```

Comment:
This is probably not a good idea, I'll remove this for now, and get a proper list of perl-modules left behind in another PR